docs/documentation/make-first-prototype/show-users-answers.md

```diff
@@ -1,55 +1,111 @@
+## Show the answer to question 1
 
-Check you’ve cleared the data by returning to a previously-loaded page and making sure the data is gone.
+1. Open `check-your-answers.html` in your `app/views` folder.
+2. Find the `<dt>` tag that contains the text 'Name'.
+3. Change 'Name' to 'Number of balls you can juggle'.
+4. In the `<dd>` tag on the next line, change 'Sarah Philips' to `{{ data['how-many-balls'] }}`.
 
-## Showing data
-
-To display user data on a different page, use this [Nunjucks](https://mozilla.github.io/nunjucks/) code:
+This is how we show data a user has entered – 'how-many-balls' is the `name` attribute from the `<input>` on the question page.
 
+Update the screen reader text – change this
 ```
-{{ data['INPUT-ATTRIBUTE-NAME'] }}
+<span class="govuk-visually-hidden"> name</span>
 ```
-
-Change `INPUT-ATTRIBUTE-NAME` to the value you used in the [`name` attribute on the question page](/docs/make-first-prototype/add-questions#add-a-text-input-to-question-2). For example:
-
+to
 ```
-{{ data['how-many-balls'] }}
+<span class="govuk-visually-hidden"> number of balls you can juggle</span>
 ```
 
-### Show the answer to question 1
+Screen readers will read this out but it will not appear on the page. Without this hidden text, screen reader users would just hear “Change” and not know what it’s for.
 
-1. Open the `check-your-answers.html` file in your `app/views` folder.
-2. Find the `<dt>` tag that contains the text `Name`.
-3. Change `Name` to `Number of balls you can juggle`.
-4. In the `<dd>` tag on the next line, change `Sarah Philips` to `{{ data['how-many-balls'] }}`.
+## Show the answer to question 2
 
-You must also change `<span class="govuk-visually-hidden"> name</span>` to `<span class="govuk-visually-hidden"> number of balls you can juggle</span>`.
+1. Find the `<dt>` tag that contains the text 'Date of birth'.
+2. Change 'Date of birth' to 'Your most impressive juggling trick'.
+3. In the `<dd>` tag on the next line, change '5 January 1978' to `{{ data['most-impressive-trick'] }}`.
 
-Screen readers will read the text in the `<span>` tags, but the text will not appear on the page.
-
-### Show the answer to question 2
+Change
+```
+<span class="govuk-visually-hidden"> date of birth</span>
+```
+to
+```
+<span class="govuk-visually-hidden"> your most impressive juggling trick</span>
+```
+Go to http://localhost:3000/start and answer the questions to check your answers show up correctly.
 
-1. Find the `<dt>` tag that contains the text `Date of birth`.
-2. Change `Date of birth` to `Your most impressive juggling trick`.
-3. In the `<dd>` tag on the next line, change `5 January 1978` with `{{ data['most-impressive-trick'] }}`.
+## Delete the remaining example answers
 
-You must also change `<span class="govuk-visually-hidden"> date of birth</span>` to `<span class="govuk-visually-hidden"> your most impressive juggling trick</span>`.
+There are example answers on the ‘Check your answers’ template page that you do not need.
```

Comment:
Is is the example answers or the template page that the user does not need? If the former, then we could always reword this as "On the 'Check your answers' template page, there are example answers that you do not need."

Reply:
former and yes that sounds good will update